packages/taquito-michelson-encoder/src/tokens/comparable/timestamp.ts

```diff
@@ -34,6 +34,10 @@ export class TimestampToken extends ComparableToken {
     return { string: val };
   }
 
+  public TypecheckValue(_val: unknown) {
+    return
```

Comment:
We should look at how timestamp values are represented by the RPC, and add an appropriate validation for val.